README.md

```diff
@@ -1,79 +1,119 @@
+### Install & Run
 
-The project is only a library yet. It means that there is no binary provided yet. To get started, you can check the examples wich are made to work with the data located in the `datasets/` folder.
+```bash
+cargo run --release
+```
 
-MeiliDB will be a binary in a near future so you will be able to use it as a database out-of-the-box. We should be able to query it using HTTP. This is our current goal, [see the milestones](https://github.com/meilisearch/MeiliDB/milestones). In the end, the binary will be a bunch of network protocols and wrappers around the library - which will also be published on [crates.io](https://crates.io). Both the binary and the library will follow the same update cycle.
+The search engine is now listening on: `http://127.0.0.1:8080`. 
 
+> You can check with `curl -i http://127.0.0.1:8080/health`
 
+### Index documents
 
-## Performances
+Create an [index](#https://docs.meilisearch.com/indexes.html) without defining the [document](#https://docs.meilisearch.com/documents.html) [schema](#to_do_schema).
+```bash
+curl --request POST 'http://127.0.0.1:8080/indexes/myindex'
+```
 
-With a database composed of _100 353_ documents with _352_ attributes each and _3_ of them indexed.
-So more than _300 000_ fields indexed for _35 million_ stored we can handle more than _2.8k req/sec_ with an average response time of _9 ms_ on an Intel i7-7700 (8) @ 4.2GHz.
+Add documents and [learn how to format your documents](#to_do_format).
 
-Requests are made using [wrk](https://github.com/wg/wrk) and scripted to simulate real users queries.
 
-```
-Running 10s test @ http://localhost:2230
-  2 threads and 25 connections
-  Thread Stats   Avg      Stdev     Max   +/- Stdev
-    Latency     9.52ms    7.61ms  99.25ms   84.58%
-    Req/Sec     1.41k   119.11     1.78k    64.50%
-  28080 requests in 10.01s, 7.42MB read
-Requests/sec:   2806.46
-Transfer/sec:    759.17KB
+```bash
+curl --request POST 'http://127.0.0.1:8080/indexes/myindex/documents' \
+  --header 'content-type: application/json' \
+  --data @movies.json
 ```
 
-### Notes
+You [can track updates](#to_do_updates) with the provided update id's .
 
-With Rust 1.32 the allocator has been [changed to use the system allocator](https://blog.rust-lang.org/2019/01/17/Rust-1.32.0.html#jemalloc-is-removed-by-default).
-We have seen much better performances when [using jemalloc as the global allocator](https://github.com/alexcrichton/jemallocator#documentation).
+### Search 
+Now that our movie dataset has been indexed, you can try out the search engine with, for example, `botman` as a query.
+```bash
+curl 'http://127.0.0.1:8080/indexes/myindex/search?q=botman'
+```
 
-## Usage and examples
+```
```

Comment:
```suggestion
```json
```

Reply:
I cannot accept this change request since the json is not well formated and the errors are displayed when with the json tag.

Reply:
Would be better to reduce the length of those lines by adding `...` or similar like the crop setting of the server do.